test/src/tests-identity.ts

```diff
@@ -398,15 +560,16 @@ describe('identity', function() {
 
         mParticle.init(apiKey, window.mParticle.config);
 
-        const cookies1 = mParticle.getInstance()._Persistence.getLocalStorage();
+            const cookies1 = mParticle
+                .getInstance()
+                ._Persistence.getLocalStorage();
         cookies1.cu.should.equal(testMPID);
         cookies1[testMPID].should.have.property('csd');
 
-        mockServer.respondWith(urls.login, [
-            200,
-            {},
-            JSON.stringify({ mpid: 'otherMPID', is_logged_in: false }),
-        ]);
+            fetchMockSuccess(urls.login, {
```

Comment:
generally i think when I see one of these `fetchMockSuccess` being used, there should be a `waitForCondition(hasLoginReturned)` following the login call below, with the tests inside of the the new `.then `.